src/main/resources/testchipip/csrc/testchip_tsi.h

```diff
@@ -6,6 +6,12 @@
 #include <fesvr/tsi.h>
 #include <fesvr/htif.h>
 
+struct init_access_t {
+  uint64_t address;
+  uint32_t stdata;
```

Comment:
Why is data 32 bits only? Is it possible to support 8/16/32/64 bit data?

Reply:
I'll leave this to future work.